trunk/SUAVE/Methods/Missions/Segments/Climb/Linear_Speed_Constant_Rate.py

```diff
@@ -40,7 +39,7 @@ def initialize_conditions(segment,state):
     v_x   = np.sqrt( v_mag**2 - v_z**2 )
     
     # pack conditions    
-    conditions.frames.inertial.velocity_vector[:,0] = v_x
+    conditions.frames.inertial.velocity_vector[:,0] = v_x[:,0]
```

Comment:
Have you checked that v_x[:,0] and v_z (without the brackets) give the same format?